docs/user_interface.rst

```diff
@@ -25,284 +25,416 @@ If you do processing or acquisition which produces new data, its associated disp
 
 .. _Calibrations Inspector Section:
 
 Calibrations
-++++++++++++
-.. for each data item
-.. offset, scale, units for each dimension
-.. displayed units
+------------
+Calibrations allow you to make specific changes to the scale and position of a selected item. Images and line plot displays have different features in the calibrations subsection.
+
+Image Calibrations
+++++++++++++++++++
+When an image is selected, the calibrations subsection of the inspector panel will show variables specific to an image.
+
+.. image:: graphics/inspector_calibrations_subsection_image.png
+    :width: 321
+    :alt: Inspector Calibrations Subsection with Image Selected
+
+With an image selected, use the calibrations subsection to
+
+* Change the offset, scale, and units on the y and x axes. The default units for images is nanometers (nm).
+
+* Change the intensity and scale of the selected image.
+
+* Change the coordinate system. This will also change how the cursor position over a display item is shown in the Info panel. See :ref:`data-items` for more information.
+
+Line Plot Calibrations
+++++++++++++++++++++++
+When a line plot is selected, the calibrations subsection of the inspector panel will show variables specific to a line plot.
+
+.. image:: graphics/inspector_calibrations_subsection_line_plot.png
+    :width: 321
+    :alt: Inspector Calibrations Subsection with Line Plot Selected
 
-For each data item displayed with the display item, the :guilabel:`Calibrations` inspector subsection shows you the dimensional and intensity calibrations and allows you to edit them. You can edit the offset, scale, and units for each dimension.
+With a line plot selected, use the calibrations subsection to
 
-.. see https://github.com/nion-software/nionswift/issues/300
+* Change the offset, scale, and units of the x axis (Channel).
 
-The :guilabel:`Display` combo box also allows you to select how the units are displayed and edited.
+* Change the coordinate system. This will also change how the cursor position over a display item is shown in the Info panel. See :ref:`data-items` for more information.
 
 .. _Session Inspector Section:
 
 Session
-+++++++
-.. for each data item
-.. specific to data item
+-------
+The Session subsection of the inspector allows you to change the session info for the selected item. 
 
-For each data item displayed with the display item, the :guilabel:`Session` inspector subsection allows you to see and edit the session info for that particular data item. For editing the global session information which gets applied to new data, see `Sessions`_.
+.. image:: graphics/inspector_session_subsection.png
+    :width: 321
+    :alt: Inspector Session Subsection
+
+Editing session info in the inspector will not change global session info. To change global session info, use the Session utility panel.
```

Comment:
Global session info gets added to each data item at acquisition. This inspector is to edit the session info for this particular data item. It might be worth explaining that here. If you have a question about this, let me know